code/modules/mining/mine_items.dm

```diff
@@ -243,7 +243,9 @@
 	threshhold.nitrogen = 82
 	threshhold.carbon_dioxide = 0
 	threshhold.toxins = 0
-	L.contents += threshhold
+	var/area/ZZ = get_area(threshhold)
+	if(!(istype(ZZ, /area/shuttle)))

```

Comment:
This is a blacklist, just with only one item
Anyway, a blacklist is fine for this. We don't typically add areas that would need to be blacklisted.